pom.xml

```diff
@@ -112,6 +112,7 @@
         <failsafe.version>1.0.4</failsafe.version>
         <redisq.version>0.0.7-SNAPSHOT</redisq.version>
         <reactivex.version>1.3.2</reactivex.version>
+        <sigar.version>1.6.4</sigar.version>
```

Comment:
Is this dependency still used somewhere? Just asking because I see this previous [commit](https://github.com/graknlabs/grakn/pull/2542/commits/7e6a7e2e4bf785082f2fa3b2e589c5be7f373165)

Reply:
@marco-scoppetta good catch!

Reply:
Fixed in https://github.com/graknlabs/grakn/pull/2542/commits/f0fbe83e06e503067bd2fdb5faec57cecf2e03b1